activitysim/benchmarking/benchmarks/mtc1mp4.py

```diff
@@ -0,0 +1,81 @@
+from activitysim.benchmarking.componentwise import (
+    template_setup_cache,
+    template_component_timings_mp,
+)
+
+import multiprocessing
+import numpy as np
+
+PRETTY_NAME = "MTC1_MP4"
+EXAMPLE_NAME = "example_mtc_full"
+NUM_PROCESSORS = int(np.clip(multiprocessing.cpu_count() - 2, 2, 4))
+CONFIGS_DIRS = ("configs_mp", "configs")
+DYNAMIC_CONFIG_DIR = "bench_configs_mp"
+DATA_DIR = "data"
+OUTPUT_DIR = "output_mp"
+COMPONENT_NAMES = [
+    "school_location",
+    "workplace_location",
+    "auto_ownership_simulate",
+    "free_parking",
+    "cdap_simulate",
+    "mandatory_tour_frequency",
+    "mandatory_tour_scheduling",
+    "joint_tour_frequency",
+    "joint_tour_composition",
+    "joint_tour_participation",
+    "joint_tour_destination",
+    "joint_tour_scheduling",
+    "non_mandatory_tour_frequency",
+    "non_mandatory_tour_destination",
+    "non_mandatory_tour_scheduling",
+    "tour_mode_choice_simulate",
+    "atwork_subtour_frequency",
+    "atwork_subtour_destination",
+    "atwork_subtour_scheduling",
+    "atwork_subtour_mode_choice",
+    "stop_frequency",
+    "trip_purpose",
+    "trip_destination",
+    "trip_purpose_and_destination",
+    "trip_scheduling",
+    "trip_mode_choice",
+]
+BENCHMARK_SETTINGS = {
+    # TODO: This multiprocess benchmarking is minimally functional,
+    # but has a bad habit of crashing due to memory allocation errors on
+    # all but the tiniest of examples. It would be great to fix the MP
+    # benchmarks so they use chunking, automatically configure for available
+    # RAM, and run a training-production cycle to get useful timing results.
```

Comment:
Does this mean that no chunking is allowed and that agencies can't benchmark their full model times?